haproxy.cfg

```
@@ -66,5 +65,33 @@ frontend dockerfrontend
     http-request allow if { path,url_dec -m reg -i ^(/v[\d\.]+)?/tasks } { env(TASKS) -m bool }
     http-request allow if { path,url_dec -m reg -i ^(/v[\d\.]+)?/version } { env(VERSION) -m bool }
     http-request allow if { path,url_dec -m reg -i ^(/v[\d\.]+)?/volumes } { env(VOLUMES) -m bool }
+    # POST requests
+    http-request allow if { path,url_dec -m reg -i ^(/v[\d\.]+)?/containers/create } { env(CONTAINERS_CREATE) -m bool }
```

You probably need this in all these rules:

```suggestion
    http-request allow if METH_POST { path,url_dec -m reg -i ^(/v[\d\.]+)?/containers/create } { env(CONTAINERS_CREATE) -m bool }
```

Also, you should modify the README explaining that the `POST` variable is global, while the other ones in the table are specific and can still allow POST requests when `POST=0`

---

> can still allow POST requests when POST=0
This seems like undesired behaviour, though I'm not sure how to make this properly. I don't recall specifics of haproxy configuration now :)